tests/integration/test_checking_s3_blobs_paranoid/test.py

```diff
@@ -689,3 +689,15 @@ def test_no_key_found_disk(cluster, broken_s3):
         "DB::Exception: The specified key does not exist. This error happened for S3 disk."
         in error
     )
+
+    s3_disk_no_key_errors_metric_value = int(
+        node.query(
+            """
+            SELECT value
```

Comment:
may be `sum(value)` otherwise you could have several integers values as a result.

Reply:
There can only be one row in the result because there's only one row in `system.metrics` where `metric = 'S3DiskNoKeyErrors'` if I understood you correctly.

Reply:
You are right.